sqlstore/src/main/resources/db/migration/V0020__qcpassed_type.sql

```diff
@@ -0,0 +1,17 @@
+ALTER TABLE Library ADD COLUMN qcPassedNew BOOLEAN NULL;
+UPDATE Library Set qcPassedNew = TRUE WHERE qcPassed = 'true' OR qcPassed = '1';
+UPDATE Library Set qcPassedNew = FALSE WHERE qcPassed = 'false' OR qcPassed = '0';
+ALTER TABLE Library DROP COLUMN qcPassed;
+ALTER TABLE Library CHANGE qcPassedNew qcPassed BOOLEAN NULL;
+
+ALTER TABLE Pool ADD COLUMN qcPassedNew BOOLEAN NULL;
+UPDATE Pool Set qcPassedNew = TRUE WHERE qcPassed = 'true' OR qcPassed = '1';
+UPDATE Pool Set qcPassedNew = FALSE WHERE qcPassed = 'false' OR qcPassed = '0';
+ALTER TABLE Pool DROP COLUMN qcPassed;
+ALTER TABLE Pool CHANGE qcPassedNew qcPassed BOOLEAN NULL;
+
+ALTER TABLE Sample ADD COLUMN qcPassedNew BOOLEAN NULL;
```

Comment:
BOOLEAN get translated to a TINYINT column like on ``Sample.emptied``, but we also have BIT columns elsewhere like on ``SampleAdditionalInfo.emptied``.  Not a dealbreaker for this PR, but worth settling on something consistent.